### PR TITLE
Fix log formatting; improving machine readability

### PR DESCRIPTION
**Pull request by SimonGLM**
Fixing some issues I encountered while reading log and data files into `pd.DataFrame`, where two whitespaces in the header lead to a column names getting assigned to the wrong data.

### CODE REVIEW

**Review by hangeza — APPROVED** [review]

**Review by hangeza — COMMENTED** [review]
Please check, whether the now enquoted string in commit 639008ceaacd498ce272004823c3eb0b7a945d89 does not confuse the DNP when parsing on the server side. Start up one detector with this code and remember the time and the detector ID to check in the database afterwards.

File: library/include/ublox_structs.h [hunk 1]
```diff
@@ -43,6 +43,7 @@ struct FixType {
         last = Undefined
     };
     static constexpr std::array<const char*, last + 1> name { "No Fix", "Dead Reck.", "2D-Fix", "3D-Fix", "GPS+Dead Reck.", "Time Fix", " N/A" };
+    static constexpr std::array<const char*, last + 1> name_log_safe { "NoFix", "DeadReck", "2D", "3D", "GPS+DeadReck", "TimeFix", "N/A" };
```
Comment (hangeza) [inline library/include/ublox_structs.h]: Pls. replace the old array with the new one instead of adding it

**Review by hangeza — APPROVED** [review]

File: library/include/ublox_structs.h (hunk 1 above)
Comment (SimonGLM) [reply library/include/ublox_structs.h]: The original array is still used in the GUI, where human readability is more important.
Should it still be removed entirely?

Comment (hangeza) [reply library/include/ublox_structs.h]: I would suggest so. The new strings are sufficiently self-explanatory to be shown in user interface contexts.

**Pull request merged**